### PR TITLE
6.2.0.1 area code 629 support

### PR DESCRIPTION
**Pull request by XXChester**
Hack to allow customers to call 629, a permanent release will come shortly to truly support this, and other missing area codes.

### CODE REVIEW

File: src/libphonenumber/PhoneNumberUtil.php [hunk 1]
```diff
@@ -679,6 +679,33 @@ public function loadMetadataFromFile($filePrefix, $regionCode, $countryCallingCo
...
     }
 
+     /**
+     * Determines if a phone number is part of a special area code that bypasses logic
+     * @param $phoneNumber
+     * @return bool if true, the number is special and special action should be taken. False otherwise.
+     */
+    public static function areWeSpecialNumber($phoneNumber)
+    {
+        return preg_match("!\\+*[\\d]{10,11}!", $phoneNumber) > 0 ? true : false;
```
Comment (MrMurga) [inline src/libphonenumber/PhoneNumberUtil.php]: I think that we need to add explicit checks for the area codes that we want to support. This check as is, and how it's used in TN server will allow non-north american numbers with digits between 10, 11 digits to be treated as a good number.

Comment (chrisleavoy) [inline src/libphonenumber/PhoneNumberUtil.php]: this doesn't look right.

**Comment by MrMurga** [comment]
👍

**Comment by chrisleavoy** [comment]
💩
 👍 
lol

**Pull request merged**